lib/flutter.yaml

```diff
@@ -0,0 +1,19 @@
+# Lints recommended for Flutter apps and packages.
+#
+# This file is meant to be used in combination with the lints defined in
+# core.yaml and recommend.yaml. Consider importing all three files in your
+# project's analysis_options.yaml file.
```

Comment:
Unfortunately, you can only import one file which makes this scenario tricky.  (This limitation helps keep options merging semantics sane in the general case -- -fyi @bwilkerson.)  I'm inclined to say `flutter.yaml` should include `core.yaml` but that does make graduating to the recommended set awkward...

Reply:
Ah! Will `recommend.yaml` import `core.yaml`? I think we would want to import `recommend.yaml` here then...

Reply:
>  Will recommend.yaml import core.yaml

Yep.

Reply:
Will Flutter want to build on recommended or core? If recommended than recommended.yaml will include core.yaml, and flutter.yaml will include recommended.yaml.

Reply:
We would want to build on recommend. I updated the file.